examples/generative/corrdiff/training/training_loop.py

```diff
@@ -177,15 +177,19 @@ def training_loop(
             )
         net_reg = Module.from_checkpoint(regression_checkpoint_path)
         net_reg.eval().requires_grad_(False).to(device)
-        interface_kwargs = dict(
-            regression_net=net_reg,
-            img_shape_x=img_shape_x,
-            img_shape_y=img_shape_y,
-            patch_shape_x=patch_shape_x,
-            patch_shape_y=patch_shape_y,
-            patch_num=patch_num,
-            hr_mean_conditioning=hr_mean_conditioning,
-        )
+
+        if loss_kwargs["class_name"] == "modulus.metrics.diffusion.ResLoss":
```

Comment:
Just checking that `class_name` will always be in `loss_kwargs` (so you don't get `KeyError`)?

Reply:
Yes, it will be always in the loss kwargs.